### DISCUSSION TITLE
Standardise error and help message

### DISCUSSION THREAD

**Discussion by jyztintan** [AY2324S2-CS2103T-W10-1/tp#174] [Announcements]
Currently in our implementation, message usage is part of the error msg that is outputted when command has an invalid format (for eg just calling an empty command `add`). 

Message help is in the callout message when calling `help` command.

Should we standardise these 2 in our error message output/calling help command

**Comment by UdhayaShan1** [comment]
By standardise you mean make them the same?

**Reply by jyztintan** [reply]
yep, so that our parser errors will give the same message as help command (but still has the "Invalid Command Format".
Also removes one "unnecessary" extra variable

**Reply by jyztintan** [reply]
for ref
before:
<img width="1343" alt="image" src="https://github.com/AY2324S2-CS2103T-W10-1/tp/assets/98398940/507616b4-b2eb-4f8a-87e7-c9573c6b6bfa">

after:
<img width="1307" alt="image" src="https://github.com/AY2324S2-CS2103T-W10-1/tp/assets/98398940/35ff859d-020a-482b-922d-0fef91920d9c">
